stdlib/2and3/logging/__init__.pyi

```diff
@@ -366,6 +366,8 @@ if sys.version_info >= (3,):
 
 
 class StreamHandler(Handler):
+    stream = ...  # type IO[str]
+    terminator = ...  # type: str
```

Comment:
This one doesn't exist in 2.7: https://github.com/python/cpython/blob/2.7/Lib/logging/__init__.py, but it's there in 3.3, the lowest Python 3 version we care about: https://github.com/python/cpython/blob/3.3/Lib/logging/__init__.py.

Can you add an `if sys.version_info >= (3, 0):` check?

Reply:
Yes, makes sense. I used `if sys.version_info >= (3,):` for consistency with the rest of the file.